style.css

```diff
@@ -1,13 +1,11 @@
 body {
   margin: 0;
-  padding: 0;
   display: block;
   background: #fff;
   font-family: 'Inter', sans-serif;
 }
 
 header {
```

Comment:
- [ ] you currently don't have a styling for the `nav` tag which holds the hamburger icon and as such the image below is what you have as the nav being on the edge of the right part of your web page, please kindly give it some margin so it can match the figma template :+1: 
![s-6-3](https://user-images.githubusercontent.com/89055502/235274674-f9fdb9ce-25fb-4a2d-8a9c-fa9eaa216638.png)

- [ ] make it bigger to match the figma template below
![figma-6](https://user-images.githubusercontent.com/89055502/235274739-a44fbb1c-2ac6-4cbf-acac-12422e306593.png)